src/ptwt/_util.py

```diff
@@ -29,6 +31,15 @@ def __len__(self) -> int:
         return len(self.dec_lo)
 
 
+WaveletDetailTuple2d = tuple[torch.Tensor, torch.Tensor, torch.Tensor]
```

Comment:
it's useful to add a docstring to each of these tuples to give some additional context about what they are and where they're used

Reply:
also, this could be a place to use a namedtuple to make it even more explicit

Reply:
Thats a good idea :+1: